### PR TITLE
Merge to master

### PR DESCRIPTION
**Pull request by alalonde**

### CODE REVIEW

**Comment by vidvidvid** [comment]
Imo we should wait for at least the following PRs to be merged into `develop` before merging `develop` -> `master`:
- https://github.com/MetaFam/TheGame/pull/959
- https://github.com/MetaFam/TheGame/pull/939
- https://github.com/MetaFam/TheGame/pull/901
- https://github.com/MetaFam/TheGame/pull/960

**Review by mquellhorst — APPROVED** [review]

**Pull request merged**
